docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.pb.bal

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We do not add licence header to BBEs. Shall we remove this?

Reply:
Fixed inace149f